### ISSUE TITLE
fix RTD builds

### ISSUE THREAD

**Issue by tomkralidis** [geopython/pycsw#521]
pycsw does seem to fail lately on our readthedocs setup.

**Comment by tomkralidis** [comment]
Found the problem. If RTD finds a `requirements.txt` file, it is used (see https://github.com/rtfd/readthedocs.org/issues/2070#issuecomment-196978116).  If in RTD Advanced Settings you specify an alternate requirements file, that is used instead.  

Since our `requirements.txt` file contains packages with C deps, RTD naturally does not find these libs and the build fails.  It seems that even if the RTD admin option "Install Project" is unchecked the problem still remains.

We ran into this problem previously, and then used `requirements-dev.txt` which had no C deps until moving things to the py.test framework.  Now, our `requirements-dev.txt` is chained to `requirements.txt` (see https://github.com/geopython/pycsw/blob/master/requirements-dev.txt#L1-L3) and the Python packages with C deps are thus installed, which is great, but breaks RTD.

@ricardogsilva is there any way we can update `requirements-dev`.txt such that it does not include other requirements files?  Or some other workaround? Note this is also causing problems as per #497.
